checkstyle.properties

```diff
@@ -1,14 +1,20 @@
 category=External Analysers
 description=Provide Checkstyle rules for Java projects
 homepageUrl=https://github.com/checkstyle/sonar-checkstyle
-archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1,10.14.2,10.15.0,10.16.0
-publicVersions=10.17.0
+archivedVersions=10.5,10.6.0,10.7.0,10.8.1,10.9.3,10.11.0,10.12.0,10.12.1,10.12.3,10.12.5,10.14.1,10.14.2,10.15.0,10.16.0,10.17.0
+publicVersions=10.19.0
 
 defaults.mavenGroupId=org.sonarsource.checkstyle
 defaults.mavenArtifactId=sonar-checkstyle-plugin
 
+10.19.0.description=Upgrade to Checkstyle 10.19.0
+10.19.0.sqVersions=[9.9,LATEST]
+10.19.0.date=2024-10-27
+10.19.0.changelogUrl=https://github.com/checkstyle/sonar-checkstyle/releases/tag/10.19.0
```

Comment:
I downgraded your 10.17 version last week based on community reports of problems & I don't see in your changelog that that has been addressed...?
https://community.sonarsource.com/t/inputmismatchexception-by-checkstyle-since-sonar-update/128807

Reply:
this issue was raised here: https://github.com/checkstyle/sonar-checkstyle/issues/533

i have tested the code part using SQ 10.7 (containerized) and cannot reproduce it. However, my analysis of multiple projects using 10.7 and the plugin works.

the assumption is that the underlying infrastructure or configuration causes this issue.
looking at the output of the issue author, the analysis uses the `windows-1252` charset which doesn't include german special characters.